ntoskrnl/mm/ARM3/procsup.c

```diff
@@ -957,6 +1033,20 @@ MmInitializeProcessAddressSpace(IN PEPROCESS Process,
     /* Release PFN lock */
     MiReleasePfnLock(OldIrql);
 
+#ifdef _M_AMD64
+   }
```

Comment:
Since this is decently far away from the opening ifdef block, it might make sense to repeat the comment here?
`/* On x64 the PFNs for the initial process are already set up */`